README.md

```diff
@@ -7,7 +7,7 @@ Manage services and containers with podman
 Usage
 -----
 
-By default this role will simply install podman and perform any other required
+By default this role will simply install podman as described in the [podman docs](https://podman.io/getting-started/installation.html) and perform any other required
 setup to make the installation functional on the supported platforms. 
```

Comment:
```suggestion
By default this role will simply install podman as described in the 
[podman docs](https://podman.io/getting-started/installation.html) and perform any other required 
setup to make the installation functional on the supported platforms.

**Important Note:** This means that for Ubuntu / Debian systems it will perform an `apt upgrade` as per the docs after the repository is added. This upgrade will only be run if the repository is added or changed and will not execute on subsequent runs.
```